internal/cluster/query.go

```diff
@@ -27,18 +28,37 @@ func QueryCluster(ctx context.Context, OrganizationSlug string, ClusterID string
 		Queues:           []Queue{},
 	}
 
+	queueChannel := make(chan Queue, len(q.Organization.Cluster.Queues.Edges))
+	errorChannel := make(chan error, len(q.Organization.Cluster.Queues.Edges))
```

Comment:
This error channel doesn't seem to ever be read from?

Reply:
What would be the best place to send this information?

Reply:
Used errgroup to handle this